packages/headless/src/app/search-engine/search-engine.ssr.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * Utility functions to be used for Server Side Rendering.
  */
-import {UnknownAction} from '@reduxjs/toolkit';
+import {AnyAction, UnknownAction} from '@reduxjs/toolkit';
```

Comment:
Same thing here about AnyAction. It seem to be deprecated and `UnknownAction ` is the exact same thing in the source code.